RMSD.py

```diff
@@ -148,16 +168,24 @@ def RMSD(ref_mol2, query_mol2):
                 # write mol2 file that does not exist into a file
                 elif value == -2:
                     oechem.OEThrow.Warning("Unable to locate %s. Skipping." % query_file)
-                    errFile.write("This queryMol does not exist: %s\n" % query_file )
+                    errFile.write("This query molecule does not exist: %s\n" % query_file )
                     rms_list.append("NaN\t")
                 else:
                     rms_list.append("%.3e" % value)
+                    if value >= opt.cutoff:
```

Comment:
Would it be more efficient to match the if statement of line 181? `if has_outlier and (value >= opt.cutoff):`

Reply:
Oh and then only have `if is_outlier` on line 181? either way you'll be checking both at some point. I'm not sure if it affects the efficiency...

Reply:
yeah, never mind my original comment haha